### ISSUE TITLE
no results for term surrounded by single quotes under Microsoft Windows 10

### ISSUE THREAD

**Issue by Konfekt** [mhinz/vim-grepper#253]
With 

```vim
let g:grepper.prompt_quote = 2
```

under Windows 10, no results are found. That is, if the search term is surrounded by single quotes, then, under Windows 10, no grepper tool (be it git-grep, rg, ag, ...) finds any result.

Instead double quotes work fine.

Therefore

```vim
let g:grepper.prompt_quote = has('win32') ? 3 : 2
```

is to be recommended.

**Comment by Konfekt** [comment]
This is not specific to vim-grepper. Vim's `grep` with `&grepprg` set just as `grepper.tool.grepprg` also shows no results.

**Comment by blm768** [comment]
As I understand it, this is a pretty normal artifact of the "standard" quoting rules on Windows (to the extent such a standard exists).

This is also an issue for the grep operator, which seems to always single-quote the query.

**Comment by Konfekt** [comment]
Yes, single quoting on Windows gives results that have little to do with what you expect on Unix
